### PR TITLE
update the example to match prism examples structure

### PR DESCRIPTION
**Pull request by JaKXz**
As documented here: https://prismjs.com/#basic-usage

and fixes #137 a small typo :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,21 +95,23 @@ const exampleCode = `
...
-          <div {...getLineProps({ line, key: i })}>
-            {line.map((token, key) => (
-              <span {...getTokenProps({ token, key })} />
-            ))}
-          </div>
-        ))}
+      <pre style={style}>
+        <code className={className}>
+          {tokens.map((line, i) => (
+            <div {...getLineProps({ line, key: i })}>
```
Comment (dekked) [inline README.md:104]: You are missing the key to the div: `<div key={i} ...`

File: README.md [hunk 2]
```diff
@@ -95,21 +95,23 @@ const exampleCode = `
...
+              {line.map((token, key) => (
+                <span {...getTokenProps({ token, key })} />
```
Comment (dekked) [inline README.md:106]: Same thing here.

**Comment by JaKXz** [comment]
@dekked the `getLineProps` and `getTokenProps` methods actually return the `key` prop and put it on the rendered element, if I understand correctly :)

**Comment by dekked** [comment]
Oh, that's right... but I still get my linter complaining :)

**Comment by JaKXz** [comment]
I'm not sure this applies anymore... @jpdriver happy to reopen if you think it's useful!

**Pull request closed without merging**